File: jewels_and_stones/jewels.py
class Solution(object):
    def numJewelsInStones(self, jewels, stones):
        """
        :type jewels: str
        :type stones: str
        :rtype: int
        """
        Jcount = 0
        for i in jewels:
            Jcount += stones.count(i)

        return Jcount

File: jewels_and_stones/jewels.cs
public class Solution {
    public int NumJewelsInStones(string jewels, string stones) {
        int Jcount = 0;
        for(int i = 0; i < jewels.Length; i++){
            for(int j = 0; j < stones.Length; j++){
                if(jewels[i] == stones[j])
                    Jcount += 1;
            }
        }
        return Jcount;
    }
}
